halo2_backend/src/plonk.rs

```diff
@@ -230,22 +230,24 @@ impl<C: CurveAffine> VerifyingKey<C> {
     }
 
     /// Returns commitments of fixed polynomials
-    pub fn fixed_commitments(&self) -> &Vec<C> {
+    #[allow(dead_code)]
+    pub(crate) fn fixed_commitments(&self) -> &Vec<C> {
         &self.fixed_commitments
     }
 
     /// Returns `VerifyingKey` of permutation
-    pub fn permutation(&self) -> &permutation::VerifyingKey<C> {
+    #[allow(dead_code)]
+    pub(crate) fn permutation(&self) -> &permutation::VerifyingKey<C> {
         &self.permutation
     }
 
```

Comment:
Following my last comment: I would either remove these methods, or keep them `pub`.

Reply:
Hmm, how about removing the `permutation` method?
I think `permutation::VerifyingKey` should not be `pub`.

Reply:
Remove the `permutation` method in https://github.com/privacy-scaling-explorations/halo2/pull/307/commits/82ff602840780aebcb845d1266e9a93c5b423785